labs/lab6/lab6.adoc

```diff
@@ -94,20 +357,18 @@ As the root user on the OpenShift master (_master-<student_id>.labs.sysdeseng.co
 oc get nodes
 ----
 
-If successful, you should see three (3) total nodes (1 master and 2 worker nodes) with **Ready** under the _Status_ column, as opposed to (2) total nodes before (1 master and 1 worker nodes). The role the instance plays can be seen through labels asociated with each node. The master node in this environment is also acting as a scheduldable instance that hosts platform infrastructure related components, such as the router and registry. Thus it has been set with the label `type=infra`. 
+If successful, you should see three (3) total nodes (1 master and 2 worker nodes) with **Ready** under the _Status_ column, as opposed to (2) total nodes before (1 master and 1 worker nodes).
 
 Execute the following to display the nodes and their associated labels:
 
-.master$
-[source, bash]
-----
-oc get nodes --show-labels
-----
-
 Red Hat CloudForms can also be used to confirm the total number of nodes has been expanded to four.
```

Comment:
This is kind of a messy transition.  Was totally un-prepared for it.

Reply:
We can remove this. It was a carry over from the old lab